buildspecs/j9.flags

```diff
@@ -2316,4 +2316,8 @@ Pass J9PORT_SIG_OPTIONS_ZOS_USE_CEEHDLR into j9sig_set_options()  before the fir
 		<description>Compile tracegenc and run it to generate tracefiles</description>
 		<ifRemoved>Run java version of tracegenc</ifRemoved>
 	</flag>
+	<flag id="windows_rebase">
```

Comment:
This flag should be either an `uma_` or `build_` flag.

Reply:
I'll rename the flag uma_windows_rebase.

Reply:
The format in the file is `prefix_camelCaseFlag`.  Can you call this `uma_windowsRebase` to match the existing convention?